packages/commonwealth/client/scripts/models/ChainInfo.ts

```diff
@@ -311,10 +311,31 @@ class ChainInfo {
     hasHomepage,
     default_allow_permissions,
     default_deny_permissions,
+    chain_node_id,
+  }: {
+    name?: string;
+    description?: string;
+    website?: string;
+    discord?: string;
+    element?: string;
+    telegram?: string;
+    github?: string;
+    stagesEnabled?: boolean;
+    customStages?: string;
+    customDomain?: string;
+    terms?: string;
+    snapshot?: string[];
+    iconUrl?: string;
+    defaultOverview?: boolean;
+    defaultPage?: DefaultPage;
+    hasHomepage?: boolean;
+    default_allow_permissions?: bigint;
+    default_deny_permissions?: bigint;
+    chain_node_id?: string;
   }) {
     // TODO: Change to PUT /chain
     const r = await $.post(`${app.serverUrl()}/updateChain`, {
-      id: app.activeChainId(),
+      id: app.activeChainId() ?? this.id,
```

Comment:
Can we move this HTTP request into a header? Generally we shouldn't have actual logic in the models file.

Reply:
Sorry what do you mean by header? another file?

Reply:
Whoops sorry, I meant helper.

Reply:
I guess it's not a big deal for now, this can go out as-is.